other/block-pod-exec-by-namespace-label.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: deny-exec-by-namespace-label
+  annotations:
+    policies.kyverno.io/title: Block Pod Exec by Namespace Label
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/minversion: 1.4.2
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      The `exec` command may be used to gain shell access to a given Pod. While this can
+      be useful for troubleshooting purposes, it could represent an attack vector and is discouraged.
+      This policy blocks Pod exec commands based upon a Namespace label `exec=false`.
+spec:
+  validationFailureAction: audit
+  background: false
+  rules:
+  - name: deny-exec-by-ns-label
+    match:
+      resources:
+        kinds:
+        - PodExecOptions
+    context:
+    - name: nslabelexec
+      apiCall:
+        urlPath: "/api/v1/namespaces/{{request.namespace}}"
+        jmesPath: "metadata.labels.exec"   
+    preconditions:
+    - key: "{{ request.operation }}"
+      operator: Equals
+      value: CONNECT
+    validate:
+      message: Exec'ing into Pods into Namespaces protected with the label `exec=false` is forbidden.
```

Comment:
change `Exec'ing into Pods into Namespaces....` to `Executing a command in a container is forbidden for pods running in Namespaces protected with the label "exec=false"`